assets/src/js/admin/admin-scripts.js

```diff
@@ -2009,8 +2013,8 @@ const gravatar = require( 'gravatar' );
 				}
 
 				// Select the donor checkbox for which delete is clicked and others should be de-selected.
-				$( '.donor-selector' ).removeAttr( 'checked' );
-				donorSelector.attr( 'checked', 'checked' );
+				$('.donor-selector').removeAttr('checked');
+				donorSelector.prop('checked', true);
```

Comment:
@kjohnson We do not need to add a hidden input field with donor id. The issue was occurring because the donor row checkbox does not check when the admin click on a single donor delete action and because of this `donor` param does not pass to bulk delete url.

![Donors ‹ give — WordPress 2021-09-29 at 11 33 07 AM](https://user-images.githubusercontent.com/1784821/135211936-9cb0d81b-ff2e-4e86-82ad-9ef282daaf94.jpg)